### PR TITLE
neovide animation plugins / chatGPT

### PR DESCRIPTION
**Pull request by simonlearnscoding**
* I changed lazy.lua so that the plugins that conflict with neovide would only be enabled if I open nvim in the Terminal.

* I added chatgpt to the plugins

### CODE REVIEW

File: lua/user/lazy.lua [hunk 1]
```diff
@@ -47,7 +58,7 @@ return function(config)
     -- whatever else you want to override with just a simple table merge
     defaults = { lazy = true },
     install = { colorscheme = { "everforest" } },
-    checker = { enabled = true }, -- automatically check for plugin updates
+    checker = { enabled = true }, -- automatically check for plugin updatesnoice
```
Comment (luxus) [inline lua/user/lazy.lua]: ?

**Comment by Uzaaft** [comment]
TBH I'm not a fan of the changes in the PR, because I don't see what they bring to the table.
If you're a neovide user, just manually change the plugins if they infer with your workflow, just like what other's have to do. 
Edit: I do like the chatGPT stuff.

**Comment by luxus** [comment]
the gui() will check if neovide or terminal runs nvim and deactivate some plugins

**Comment by Uzaaft** [comment]
chatgpt has been added by luxus

**Comment by Uzaaft** [comment]
Same goes for the neovide stuff. Closing this PR since the changes have been implemented upstream, and the OP didn't respond to luxus in due time.

**Pull request closed without merging**